package.json

```diff
@@ -62,7 +62,7 @@
     "@github/prettier-config": "0.0.4",
     "@playwright/test": "^1.35.1",
     "@primer/css": "21.0.2",
-    "@primer/primitives": "7.12.0",
+    "@primer/primitives": "0.0.0-20230921000240",
```

Comment:
Shouldn't we fix this to a release

Reply:
Yes, I need to make another patch release before I merge this PR